docs/GroupDefs.hpp

```diff
@@ -48,6 +48,42 @@
 /*!
  * \defgroup ControlSystemGroup Control System
  * \brief Contains control system elements
+ *
+ * The control system manages the time-dependent mapping between the fixed
+ * computational frame (grid frame) and the inertial frame. The time-dependent
+ * parameters of the mapping are adjusted by a feedback control system in order
+ * to follow the dynamical evolution of the black holes. In BBH simulations the
+ * map is typically a composition of maps that include translation, rotation,
+ * scaling, shape, etc. Each map under the governance of the control system has
+ * an associated time-dependent map parameter \f$\lambda(t)\f$ that is regularly
+ * updated by a control signal \f$U(t)\f$, which is determined by an error
+ * function \f$Q(t)\f$ that is uniquely specified for each map.
```

Comment:
Add "Note that \f$\lambda(t)\$, \f$U(t)\f$, and \f$Q(t)\f$ can be vectors."

Also add something stating that the control system adjusts lambda so that Q is driven to zero.